packages/terra-table/src/subcomponents/_Cell.jsx

```diff
@@ -55,7 +55,7 @@ const Cell = ({
   ...customProps
 }) => {
   const cellClassNames = cx('cell');
-  const contentClass = !removeInner ? cx('container') : undefined;
+  const contentClass = !removeInner ? cx('container') : cx('content-width');
```

Comment:
We should probably move the 'content-width' class to:
https://github.com/cerner/terra-core/blob/master/packages/terra-table/src/subcomponents/_Cell.jsx#L75

That way this check is still valid:
```
if (ariaAttr || contentClass) {
```

It would probably look something like:
```
<div {...ariaAttr} className={contentClass || cx('content-width')}>
```

Reply:
That was my mistake prior, I didn't consider that if check with my previous advice.